source/nodejs/adaptivecards-designer/src/card-designer.ts

```diff
@@ -106,8 +110,12 @@ export class CardDesigner extends Designer.DesignContext {
                     '<div style="padding: 8px; display: flex; justify-content: center;">' +
                         '<div>The Card Structure isn\'t available in Preview mode.</div>' +
                     '</div>';
-            }
-            else {
+            } else if (this._isPayloadCardArray) {
+                this.treeViewToolbox.content.innerHTML =
+                    '<div style="padding: 8px; display: flex; justify-content: center;">' +
+                        '<div>The Card Structure isn\'t available for this payload.</div>' +
```

Comment:
Need to add the reason. For example, `The Card Structure isn\'t available for this payload because multiple adaptivecards are defined...`